plugins/chat/app/models/chat/message.rb

```diff
@@ -122,19 +123,15 @@ def validate_message
       end
     end
 
-    def excerpt(max_length: 100)
+    def raw_excerpt
       # just show the URL if the whole message is a URL, because we cannot excerpt oneboxes
```

Comment:
We could potentially move this out of the model later but I don't think it's a big issue to have it here